### PR TITLE
[7-43] modify language dropdown styles on footer

### PR DESCRIPTION
**Pull request by boburwebdev**
image

### CODE REVIEW

**Review by HrebenYury — APPROVED** [review]

File: exalearn-ui/src/app/components/footer/footer.component.scss [hunk 1]
```diff
@@ -2,14 +2,20 @@
 
 .footer {
 	width: 100vw;
-	text-align: center;
 	position: fixed;
 	bottom: 0;
-	height: $footer-height;
+	height: $footer-height + 10px;
```
Comment (mityajs) [inline exalearn-ui/src/app/components/footer/footer.component.scss]: wtf

**Review by mityajs — APPROVED** [review]

File: exalearn-ui/src/app/components/footer/footer.component.scss (hunk 1 above)
Comment (KuznetsovR) [inline exalearn-ui/src/app/components/footer/footer.component.scss]: maybe you'll just change $footer-height

Comment (boburwebdev) [reply exalearn-ui/src/app/components/footer/footer.component.scss]: tried this one first, worked. then changed footer from 30px to 40px. Styles broke, went with this one )

Comment (boburwebdev) [reply exalearn-ui/src/app/components/footer/footer.component.scss]: tried this one first, worked. then changed footer from 30px to 40px. Styles broke, went with this one )

Comment (boburwebdev) [reply exalearn-ui/src/app/components/footer/footer.component.scss]: fixed

Comment (boburwebdev) [reply exalearn-ui/src/app/components/footer/footer.component.scss]: fixed

**Review by HenadzV — APPROVED** [review]

**Pull request merged**